ui/lib/subscription/getMarketplaceListings.ts

```diff
@@ -0,0 +1,38 @@
+import { Arbitrum, Sepolia } from '@thirdweb-dev/chains'
```

Comment:
Nice! Could you also add a test for the expiration? Add a listing w/ an expiration and make sure that it renders as expired.